### ISSUE TITLE
non propriatary alternatives

### ISSUE THREAD

**Issue by Utopiah** [minosvasilias/toolformer-zero#3]
Would be nice to rely on e.g BLOOM (via HuggingFace inference API) and DuckDuckGo. The result might be qualitatively different but the principle would remain the same without the apparent need to use blackboxes.

**Comment by minosvasilias** [comment]
I agree that would be a good addition, thanks for the suggestion! Comparing performance would be interesting as well.
I'll do some research on the required APIs and see if i can implement something like that when i have time.

Otherwise, contributions are of course welcome.
